package.json

```diff
@@ -7,14 +7,30 @@
     "doc": "docs"
   },
   "dependencies": {
+    "babel-eslint": "^10.0.1",
+    "eslint": "^5.8.0",
+    "eslint-config-airbnb": "^17.1.0",
+    "eslint-config-prettier": "^3.3.0",
+    "eslint-config-standard": "^12.0.0",
+    "eslint-plugin-import": "^2.14.0",
+    "eslint-plugin-jsx-a11y": "^6.1.2",
+    "eslint-plugin-mocha": "^5.2.0",
+    "eslint-plugin-node": "^8.0.0",
+    "eslint-plugin-promise": "^4.0.1",
+    "eslint-plugin-react": "^7.11.1",
+    "eslint-plugin-standard": "^4.0.0",
     "eslint-plugin-jest": "^22.1.3",
-    "netlify-cli": "^2.6.0"
+    "link-parent-bin": "^0.2.3",
+    "netlify-cli": "^2.6.0",
+    "prettier": "^1.15.3",
+    "prettier-eslint-cli": "^4.7.1"
   },
   "devDependencies": {
-    "husky": "^1.2.0",
-    "ganache-cli": "^6.2.3"
+    "ganache-cli": "^6.2.3",
+    "husky": "^1.2.0"
   },
   "scripts": {
+    "link-parent-bin": "link-parent-bin -c . -s true",
```

Comment:
This copies the binaries from the root into the subfolder's node_modules too.

Reply:
And right now it does it in a dumb way which means we have a couple danging `node_modules` (in wiki in particular...) but I have [commented](https://github.com/nicojs/node-link-parent-bin/issues/7#issuecomment-454042779) on an issue with that project to allow us to specify which things folder we need to have this copied to.